test/test_regression.jl

```diff
@@ -177,25 +176,25 @@ function run_rayleigh_benard_regression_test(arch)
     # Model setup
     #
 
+    # Force salinity as a passive tracer (βS=0)
+    S★(x, z) = exp(4z) * sin(2π/Lx * x)
+    FS(grid, u, v, w, T, S, i, j, k) = 1/10 * (S★(grid.xC[i], grid.zC[k]) - S[i, j, k])
+
     model = Model(
          arch = arch,
             N = (Nx, Ny, Nz),
             L = (Lx, Ly, Lz),
             ν = ν,
             κ = κ,
           eos = LinearEquationOfState(βT=1., βS=0.),
-    constants = PlanetaryConstants(g=1., f=0.)
+    constants = PlanetaryConstants(g=1., f=0.),
+          bcs = BoundaryConditions(T=FieldBoundaryConditions(z=ZBoundaryConditions(

```

Comment:
@ali-ramadhan see here for how to set boundary conditions in `z` on a particular field.